src/swanctl/swanctl.opt

```diff
@@ -999,6 +1001,9 @@ connections.<conn>.children.<child>.if_id_in = 0
 	ID. The special value _%unique_ sets a unique interface ID on each CHILD_SA
 	instance, beyond that the value _%unique-dir_ assigns a different unique
 	interface ID for each CHILD_SA direction (in/out).
+	This value can be hexadecimal (start with 0x), decimal (no special prefix), or octal (start with 0).
+	The conversion of the numerical values is done using the **strtoul** function thus all quirks and
```

Comment:
That could maybe be added to the above section. But what quirks are you referring to exactly?

Reply:
Whatever the documentation for strtoul does on the system strongSwan is running. It can be different on 'BSD and Linux, as the man page for it implies. That's why that's noted here.

Reply:
Since it's a standard C function, I don't think there are major differences between platforms (the only note regarding BSD I found in the man page I have here is that `strtouq()` is provided there, but that's obviously not relevant as we don't call that function). There could be implementation-specific differences that occur in non-C or -POSIX locales, but that mainly broadens the accepted input (e.g. regarding thousands separators), so that shouldn't matter much either.

Reply:
It'd be good if the locale dependence was documented so if it ever happens to be a problem, the man page mentions it.